redis/src/types.rs

```diff
@@ -2691,6 +2693,24 @@ pub struct PushInfo {
     pub data: Vec<Value>,
 }
 
+impl PushInfo {
+    /// Attempts to parse the push message's payload into a specific type.
+    pub fn try_from_redis_value<T>(mut self) -> RedisResult<T>
```

Comment:
IMHO it doesn't make much sense to have it, since it tries from singular value. Some PushKinds (e.g. *Message) has multiple data in it.
Or you can change documentation of function about this.